content/en/blog/_posts/2022-03-31-Kubernetes-1-24-deprecations-and-removals

```diff
@@ -0,0 +1,90 @@
+---
+layout: blog
+title: "Kubernetes Removals and Deprecations In 1.24"
+date: 2022-03-31
+slug: upcoming-changes-in-kubernetes-1-24
+---
+
+**Author**: Mickey Boxell (Oracle)
+
+As Kubernetes evolves, features and APIs are regularly revisited and removed. New features may offer an alternative or improved approach to solving an existing problem, motivating the team to remove the old method. Rather than immediately being removed, old APIs are first deprecated and then removed according to the [Kubernetes Deprecation Policy](/docs/reference/using-api/deprecation-policy/). 
+
+We want to make sure you are aware of the changes coming in the Kubernetes 1.24 release. Kubernetes 1.24 is scheduled to be released in April. The release will **deprecate** several (beta) APIs that will become deprecated in favor of stable versions of the same APIs. The major change coming in the Kubernetes 1.24 release is the [removal of Dockershim](https://github.com/kubernetes/enhancements/issues/2221). This will be discussed below and in more depth at release time.  
```

Comment:
```suggestion
We want to make sure you are aware of the changes coming in the Kubernetes 1.24 release. Kubernetes 1.24 is scheduled to be released in April. The release will **deprecate** several (beta) APIs in favor of stable versions of the same APIs. The major change coming in the Kubernetes 1.24 release is the [removal of Dockershim](https://github.com/kubernetes/enhancements/issues/2221). This will be discussed below and in more depth at release time.  
```